packages/components/addon/components/hds/side-nav/index.js

```diff
@@ -56,6 +66,10 @@ export default class HdsSideNavComponent extends Component {
     return this.isResponsive && !this.isDesktop && !this.isMinimized;
   }
 
+  get showToggleButton() {
+    return (this.isResponsive && !this.isDesktop) || this.isCollapsible;
```

Comment:
The suggested alternative is not an equivalent, unfortunately, as it would show the toggle button even when `isCollapsible` is false.

It should read as: show the toggle button if the user is on mobile and `isResponsive` is set to `true' or, regardless of the viewport, if `isCollapsible` is `true`. This is the current logic we want to reflect, I'm not sure we can simplify it further 🤔